src/util/api/formatResponseObject.ts

```diff
@@ -0,0 +1,65 @@
+import { camelCase, mapKeys, pick, rearg } from "lodash";
+
+import { VenueDetails } from "../../types";
+
+const toCamelCase = (obj) => {
+  return mapKeys(obj, rearg(camelCase, 1));
+};
+
+export default function formatResponseObject(
+  obj,
+  source: string
+): VenueDetails {
+  switch (source) {
+    case "linked":
+      const linked = pick(obj, [
+        "id",
+        "data_source",
+        "email",
+        "contact_type",
+        "postal_code",
+        "image",
+        "address_locality",
+        "position",
+        "description",
+        "name",
+        "info_url",
+        "street_address",
+        "telephone",
+      ]);
```

Comment:
As a question, does linked's data model just happen to align well with the intended data model for `VenueDetails` or does `VenueDetails` copy the data model of linked?

Reply:
I took a look at responses I got from each url and decided to use linked's response as a base. Imo it made a little more sense.

Reply:
Ok. Personally I would have thought about the data model in terms of the requirements of the UI and use neither data source's model as the base. If we were to use a base, then perhaps unified search could have been a candidate, because sharing the model of the data between the search and detail views could have provided some benefits.

This is a minor detail, but as it stands now, it seems that this approach makes it likely that future developers will also copy linked's data model as is--as opposed to approaching the data model through the requirements of the UI.

Of course, linked is a well crafted service and the data model it provides is likely perfectly fine for us. As far as I can see, there's nothing I would want to change outright.

Reply:
😄  don't get me wrong, of course I gave this a thought on UI's standpoint as well. Even if I'd start all over again `VenueDetails` is like 99% something that I would have returned anyway.